doc/toxic.conf.5

```diff
@@ -2,12 +2,12 @@
 .\"     Title: toxic.conf
 .\"    Author: [see the "AUTHORS" section]
 .\" Generator: DocBook XSL Stylesheets vsnapshot <http://docbook.sf.net/>
-.\"      Date: 2024-02-17
+.\"      Date: 2024-02-09
```

Comment:
are we traveling back in time? :smile:

Reply:
I dunno, the dates are auto generated and do weird things sometimes